arbnode/node.go

```diff
@@ -44,83 +45,6 @@ import (
 	"github.com/offchainlabs/nitro/validator"
 )
 
-type RollupAddresses struct {
```

Comment:
`RollupAddresses` should stay in node.go, `L1Config` should be moved into node.go as well, that will solve the circular dependency.

Reply:
Done